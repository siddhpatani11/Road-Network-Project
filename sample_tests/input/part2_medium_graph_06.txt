37 282
0 1 30 B
0 9 33 G
0 9 27 B
0 17 44 R
0 17 41 R
0 26 13 R
0 29 26 G
0 29 16 G
2 0 4 R
2 0 42 R
2 14 5 G
2 14 32 G
2 21 18 B
2 21 40 B
2 27 36 R
2 27 19 R
3 4 35 R
3 4 46 G
3 23 19 B
4 5 18 R
4 5 36 R
4 12 8 B
4 12 21 G
4 15 26 B
4 15 49 R
4 32 30 B
5 4 14 G
5 8 23 R
5 16 20 R
5 26 3 R
5 33 17 R
5 33 18 R
6 3 39 G
6 3 41 B
6 8 33 R
6 9 50 R
6 11 34 G
6 11 18 G
6 18 15 R
6 18 25 G
6 24 9 R
6 24 36 B
6 31 19 B
6 31 35 B
6 34 48 R
6 34 6 R
7 28 6 G
7 28 14 B
7 29 12 B
7 29 3 G
8 5 26 R
8 9 15 B
8 9 8 B
8 19 48 B
8 31 13 B
8 35 41 G
8 35 27 R
9 13 46 G
9 13 41 R
9 16 36 R
9 16 30 B
9 19 7 R
9 19 28 B
9 30 2 G
10 1 42 G
10 1 48 B
10 34 20 R
11 3 4 R
11 3 38 G
11 13 14 G
11 27 26 B
11 27 5 G
11 34 38 G
11 36 49 R
11 36 15 G
12 1 18 G
12 1 3 B
12 11 49 B
12 11 49 B
12 17 8 G
12 24 32 G
12 27 39 G
12 35 31 G
12 35 8 G
13 0 8 G
13 0 46 B
13 1 28 G
13 5 7 G
13 5 12 B
13 36 13 R
14 12 12 R
14 12 30 G
14 29 32 B
14 29 10 G
15 5 50 R
15 5 49 G
15 14 2 B
15 14 10 B
15 18 21 B
15 24 49 B
15 24 9 B
15 29 1 R
15 29 21 R
15 34 27 B
15 34 48 B
15 36 33 R
15 36 32 B
16 3 35 B
16 13 11 R
16 19 23 B
16 19 2 R
16 22 24 R
16 22 27 B
17 6 50 G
17 18 4 R
17 22 50 R
17 34 15 R
17 34 26 G
18 1 6 R
18 11 5 B
18 11 20 G
18 13 46 R
18 13 33 G
18 26 24 R
18 26 19 R
18 32 22 R
18 32 33 B
18 35 28 B
18 35 21 G
19 10 3 G
19 14 45 G
19 14 10 B
19 15 33 B
19 15 49 B
19 16 8 G
19 16 26 R
19 24 29 B
19 24 33 B
19 28 9 R
19 34 49 R
20 1 24 G
20 1 12 G
20 2 1 R
20 2 16 G
20 11 47 G
20 11 2 R
20 12 35 G
20 25 36 G
20 25 24 R
20 28 42 G
20 28 13 R
20 30 51 B
20 31 17 R
21 4 39 R
21 4 49 R
21 7 42 B
21 12 32 G
22 9 13 G
22 17 42 B
22 17 16 B
22 30 42 R
22 30 2 B
23 4 2 G
23 4 39 R
23 7 48 B
23 7 28 G
23 14 22 R
23 14 40 R
23 15 45 R
23 15 37 B
23 19 4 R
23 19 16 R
23 32 13 R
23 32 31 G
24 16 16 G
24 18 46 R
24 18 16 R
24 19 28 G
24 19 11 R
24 23 14 G
24 23 6 G
24 25 38 B
24 25 42 R
24 35 8 R
24 35 4 B
25 15 14 G
25 15 34 R
25 31 19 B
25 31 19 R
26 0 7 B
26 0 42 R
26 10 43 G
26 12 35 G
26 17 46 B
26 17 23 G
26 30 13 B
27 0 48 B
27 1 20 G
27 4 20 B
27 4 30 G
27 9 20 B
27 9 40 R
27 15 20 R
27 15 27 B
27 16 5 B
27 31 33 G
28 2 42 G
28 9 3 G
28 9 25 R
28 14 50 B
28 21 3 B
29 14 22 R
29 19 21 B
29 21 5 R
29 26 36 R
29 26 41 G
29 35 8 B
29 35 17 R
30 3 20 R
30 10 43 R
30 15 16 B
30 20 49 G
30 23 33 G
30 23 47 R
30 24 8 G
30 27 14 R
31 4 23 R
31 8 12 B
31 8 17 B
31 12 23 G
31 12 6 R
31 16 11 G
31 16 26 G
31 27 44 B
31 27 50 R
31 36 22 G
32 3 9 R
32 5 44 G
32 9 45 R
32 9 36 R
32 13 50 B
32 13 43 G
32 18 27 R
32 24 42 R
32 28 22 R
32 30 16 G
32 30 39 R
33 7 42 R
33 14 15 G
33 14 37 B
33 24 24 B
33 24 41 B
34 0 31 B
34 0 45 B
34 11 11 R
34 13 26 R
34 30 16 G
34 32 6 G
34 32 33 R
34 33 22 R
34 33 5 R
35 5 33 B
35 5 5 G
35 11 44 B
35 11 24 R
35 12 50 R
35 13 13 B
35 13 9 B
35 15 19 R
35 15 18 R
35 17 18 G
35 17 27 B
35 19 44 B
35 19 7 R
35 27 51 G
35 29 50 G
36 6 43 G
36 6 11 B
36 10 48 B
36 10 33 R
36 12 3 R
36 16 6 G
