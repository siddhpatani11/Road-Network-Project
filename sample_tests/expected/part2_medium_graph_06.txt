12 G 63 B 3 G 103 B 53 G 59 G 41 G 73 G 149 G 64 G 59 G 46 B 49 - 0 G 17 G 73 G 54 G 36 G 8 G 39 G 43 G 154 G 110 G 63 G 101 G 32 G 70 G 44 G 39 G 61 G 67 G 66 G 89 G 56 G 58 G 50 G 8 G 30 
